Main/Controllers/SharedCollectionController.cs

```diff
@@ -69,5 +69,58 @@ public IActionResult ShareCollection(SharedCollectionCreationData data)
       _dbContext.SaveChanges();
       return Ok();
     }
+
+    [HttpPatch]
```

Comment:
This should be a patch to "{collectionId}", like in delete, not the whole endpoint.